mathparse/mathparse.py

```diff
@@ -96,7 +96,18 @@ def replace_word_tokens(string, language):
 
     # Replace operator words with numeric operators
     operators = words['binary_operators'].copy()
-    operators.update(words['unary_operators'])
+    if u'unary_operators' in words:
+        operators.update(words['unary_operators'])
+
+    # Ensures unicode string processing
+    import sys
+    if sys.version_info.major == 3:
+        if not isinstance(string, str):
+            string.decode("utf-8")
```

Comment:
This `decode` line might not be doing anything. I believe it [returns the decoded value](https://docs.python.org/3/library/stdtypes.html#bytes.decode) and this line doesn't assign it with `string =` like the one bellow.

Reply:
@gunthercox Good observation, I think this code is not necessary too because in python3 strings already are unicode.